libs/ui-libraries/src/lib/repo-info/repo-info.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="ml-5 justify-content-start h-100 col-2 flex-column d-flex test">
+    <div class="profile w-100 flex-column border-bottom">
+        <div class="pic"></div>
+        <h3 style="color: #5B5B5B" >John Doe</h3>
```

Comment:
We should really try to stay away from adding `style` attributes anywhere.
Always try bootstrap utility classes, then create classes. Style attributes are last resort.